context/app/static/js/index.jsx

```diff
@@ -3,6 +3,12 @@ import ReactDOM from 'react-dom';
 
 import App from './components/App';
 
+// eslint-disable-next-line no-undef
+const validation_errors = flaskData.entity?.mapper_metadata?.validation_errors;
+if (validation_errors && validation_errors.length) {
```

Comment:
```suggestion
if (validation_errors) {
```
I don't think we have to worry about a falsey value here since we're not displaying anything, but we can keep it if you want to be safe.

Reply:
(and it's also not a short circuit)